canvas.gypi

```diff
@@ -0,0 +1,43 @@
+{
+    'variables':
+    {
+        'has_FBDev%': 'true',
```

Comment:
It's hardcoded to `true` because I don't know how to be able to automatically detect that (there's no library as with X11, and for cross-builds looking at `/dev/fb0` would lead to false positives), but there's no problem on having FBDev support enabled by default just for that reason, it will only give an error when trying to access to the `/dev/fb0` if it doesn't exists or you don't have permissions :-) You can be able to explicity disable FBDev support by executing `npm install --has_FBDev=false`, and the same for X11.